jsonnetlib/k8s.jsonnet

```diff
@@ -78,6 +78,8 @@ cfg {
     volumes=[],
     podSpec=root.podSpec,
     cronjobSpec=root.cronjobSpec,
+    memoryRequest=root.cronMemoryRequest,
+    memoryLimit=root.cronMemoryLimit,
```

Comment:
After defining it here, you'll need to use the variable in the function body. Speaking of that, can you add tests to ensure it works as you expect it?

Reply:
@idealhack Sorry for the reckless pr. I fixed them and add test case now.